src/server/browserFetcher.ts

```diff
@@ -28,6 +29,8 @@ import { assert } from '../helper';
 
 const unlinkAsync = util.promisify(fs.unlink.bind(fs));
 const chmodAsync = util.promisify(fs.chmod.bind(fs));
+const mkdirAsync = fs.promises.mkdir;
```

Comment:
let's not use `fs.promises` - I think they're still experimental in node 10.

Instead, we can use good-old `util.promisify`:
```js
const mkdirAsync = util.promisify(fs.mkdir.bind(fs));
```